Makefile

```diff
@@ -8,19 +10,36 @@ freeze-requirements:
 
 .PHONY: test-black
 test-black:
-	black --check --diff .
+	black --check --diff $(SOURCES)
 
 .PHONY: test-codespell
 test-codespell:
-	codespell --quiet-level 4 --ignore-words-list crate,keyserver --skip '*.tar,*.xz,*.zip,*.bz2,*.7z,*.gz,*.deb,*.rpm,*.snap,*.gpg,*.pyc,*.png,*.ico,*.jar,changelog,.git,.hg,.mypy_cache,.tox,.venv,_build,buck-out,__pycache__,build,dist,.vscode,parts,stage,prime,test_appstream.py,./snapcraft.spec,./.direnv,./.pytest_cache'
+	codespell --quiet-level 4 --ignore-words-list crate,keyserver --skip '*.tar,*.xz,*.zip,*.bz2,*.7z,*.gz,*.deb,*.rpm,*.snap,*.gpg,*.pyc,*.png,*.ico,*.jar,*.so,changelog,.git,.hg,.mypy_cache,.tox,.venv,_build,buck-out,__pycache__,build,dist,.vscode,parts,stage,prime,test_appstream.py,./snapcraft.spec,./.direnv,./.pytest_cache'
 
 .PHONY: test-flake8
 test-flake8:
-	python3 -m flake8 .
+	python3 -m flake8 $(SOURCES)
+
+.PHONY: test-isort
+test-isort:
+	isort --check $(SOURCES)
 
 .PHONY: test-mypy
 test-mypy:
-	mypy .
+	mypy $(SOURCES)
+
+.PHONY: test-pydocstyle
+test-pydocstyle:
+	pydocstyle snapcraft
+
+.PHONY: test-pylint
+test-pylint:
+	pylint snapcraft
+	pylint tests/*.py tests/unit --disable=invalid-name,missing-module-docstring,missing-function-docstring,no-self-use,duplicate-code,protected-access,unspecified-encoding
```

Comment:
why is this not on pylintrc?

Reply:
We can change this in a future PR if needed.